Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -915,6 +1105,22 @@ static inline IntDegrees sensorOrientation(AVCaptureVideoOrientation videoOrient
     dispatchVideoFrameToObservers(WTFMove(videoFrame), metadata);
 }
 
+void AVVideoCaptureSource::captureOutputDidFinishProcessingPhoto(RetainPtr<AVCapturePhotoOutput>, RetainPtr<AVCapturePhoto> photo, RetainPtr<NSError> error)
+{
+    if (!error) {
+        NSData* data = [photo fileDataRepresentation];
+        resolvePendingPhotoRequest({ static_cast<const uint8_t*>(data.bytes), data.length }, "image/jpeg"_s);
```

Comment:
return here and no `else`.
I would tend to do `if (error)` as well instead of `if(!error)`